logging/cluster-logging-release-notes.adoc

```diff
@@ -1,19 +1,312 @@
 :_content-type: ASSEMBLY
-[id="cluster-logging-release-notes"]
+[id="cluster-logging-release-notes_{context}"]
 include::_attributes/common-attributes.adoc[]
```

Comment:
This is showing up in the previews.  
![Attributes](https://user-images.githubusercontent.com/4106682/168663466-8a20903d-61c8-4070-87dc-899bfb98ab67.png)